app/models/questionnaire.rb

```diff
@@ -0,0 +1,17 @@
+class Questionnaire < ApplicationRecord
+  self.primary_key = "version_id"
```

Comment:
Why are we calling the `id` as `version_id` here?

Reply:
Because `version_id` is a foreign key that points to `questionnaire_version`'s `id`.